internal/sci/aws/awssci.go

```diff
@@ -0,0 +1,268 @@
+// Package aws provides an AWS implementation of the Substratus Cloud Interface (SCI)
+package awssci
```

Comment:
I am just calling it `gcp`  instead of `gcpcsi`

Reply:
just calling out a difference, not sure what's recommended here ha

Reply:
Calling it aws would conflict with my imports. I thought this was the right distinction but maybe aliasing the imports is preferred

Reply:
Best to match dir and alias on import to keep with conventions, gophers like their conventions

Reply:
can do

Reply:
Ok +1 to match dit and use alias for import

Reply:
done